deploy/kubeturbo-operator/certified-operator-automation-1.py

```diff
@@ -0,0 +1,178 @@
+import os
+import shutil
+from github import Github
+import git
+import re
+import docker
+
+#icr parameters
+icr_url = 'icr.io'
+icr_namespace = 'cpopen'
+#Enter the local repo path to clone repository to modify files
+local_repo_path = os.getenv('LOCAL_REPO_PATH')
+if local_repo_path is None:
+    local_repo_path = input('Enter repository local path: ')
+#PR parameters
+base_repo_name = 'certified-operators'
+base_repo_owner = 'redhat-openshift-ecosystem'
+origin_repo_name = 'certified-operators'
+origin_repo_owner = 'turbodeploy'
+#Git access token
+access_token = os.getenv('ACCESS_TOKEN')
+if access_token is None:
+    access_token = input('Enter turbodeploy access token: ')
+repo_url = f'https://turbodeploy:{access_token}@github.com/turbodeploy/certified-operators.git'
+
+# Clone the repository to a local directory
+git.Repo.clone_from(repo_url, local_repo_path)
+
+#To keep the code always in sync with base repository of redhat-openshift-ecosystem/certified-operators
+#before releasing any new operator
+# Add the base repository as a remote
+repo = git.Repo(local_repo_path)
+base_url = f'https://github.com/{base_repo_owner}/{base_repo_name}.git'
+repo.create_remote('base', base_url)
+# Fetch the changes from the base repository
+base = repo.remote(name='base')
+base.fetch()
+# Merge the changes into the local main branch
+repo.git.merge('base/main')
+# Push the changes to the forked repository
+origin = repo.remote(name='origin')
+origin.push()
+
+# Create Operator version folder
+operator_name = input('Enter the name of certified operator to release (kubeturbo or prometurbo):')
+operator_path = f'/operators/{operator_name}-certified'
+operator_dir_path = local_repo_path + operator_path
+operator_release_version = input(f'Enter the version of the {operator_name} certified operator to release: ')
```

Comment:
Let's take all user's input from one place. I recommend at the beginning like a pre-requisite to run the script.